torchaudio/prototype/models/hifi_gan.py

```diff
@@ -334,3 +335,85 @@ def hifigan_generator_v3() -> HiFiGANGenerator:
         in_channels=80,
         lrelu_slope=0.1,
     )
+
+
+class HiFiGANMelSpectrogram(torch.nn.Module):
```

Comment:
Let's not have this as a public API yet.
Please move it to bundle implementation and prefix with underscore.

Reply:
Done